ddtrace/tracer/context_test.go

```diff
@@ -107,6 +110,40 @@ func TestStartSpanFromContextRace(t *testing.T) {
 	assert.ElementsMatch(t, outputs, expectedTraceIDs)
 }
 
+func Test128(t *testing.T) {
+	_, _, _, stop := startTestTracer(t)
+	defer stop()
+
+	span, _ := StartSpanFromContext(context.Background(), "http.request")
+	assert.NotZero(t, span.Context().TraceID())
+	if w3cCtx, ok := span.Context().(ddtrace.SpanContextW3C); ok {
```

Comment:
super nit but on re-reading this can we flip the condition here to push the "happy-path" of the test to the left? (And on the `if` below on 134 too?